css/style.css

```diff
@@ -449,3 +424,175 @@ footer {
   align-items: center;
   padding-left: 0;
 }
+
+/* media querry for small screens */
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] I think this media query is messing up your design at `768px`, the headline is in mobile view and the rest is in desktop view, the result looks weird and not coherent 👇 It's better if you make the mobile version without the use of media queries then for desktop version you use media queries and make the necessary changes 👍 

## Your design:
![Animation](https://user-images.githubusercontent.com/50721479/213428879-28c24a09-2b07-4974-82bc-34b0c46a512a.gif)